Sources/Components/ReviewButtonView/ReviewButtonView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReviewButtonViewDelegate: AnyObject {
+    func reviewButtonView(_ reviewButtonView: ReviewButtonView, giveReviewWasTapped startReview: Bool)
+}
+
+public final class ReviewButtonView: UIView {
+    // MARK: - Public
+
+    public weak var delegate: ReviewButtonViewDelegate?
+    public var text: String = "" {
```

Comment:
Since this class is `ReviewButtonView` and not `ReviewButton` it might make sense to rename this to `buttonText` or `buttonTitle`. I think it would make the intent of the api clearer.